staging/prometheus-thanos/crds/crd-alertmanager.yaml

```diff
@@ -0,0 +1,4481 @@
+# https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml
```

Comment:
Please follow these guidelines for CRDs. Yes, it's only documented on my blog. I'll pull that content into a document in the charts repo, but for now: https://joejulian.name/post/helm-v2-crd-management/

Reply:
@joejulian can you please be more specific what changes you request to do? To my understanding what you request is already there. https://github.com/mesosphere/charts/commit/4982c96dd316f2f93cefd9eab61b67f4205a0a95

Reply:
Resolving this as we are not modifying `prometheus-operator` anymore but have it as a `subchart`.